setup.py

```diff
@@ -41,11 +41,23 @@
 #                                                                              #
 ################################################################################
 
-import setuptools
+import sys
 import textwrap
 
+import setuptools
+
+
 version = "1.43.8"
 
+tests_require = [
+    "cryptography",
+    "httpretty==0.9.6",
+    "parameterized==0.7.0",
+]
+
+if sys.version_info < (3, 3):
+    tests_require.append("mock==3.0.5")
```

Comment:
Part of me does not like this duplication. However I just merged a change changing the httpretty requirement. Could you change it to >=? And I note I didn't fix test-requirements, sigh.